Formula/hypre.rb

```diff
@@ -14,20 +15,10 @@ class Hypre < Formula
 
   depends_on "gcc" # for gfortran
   depends_on "open-mpi"
-  depends_on "veclibfort"
 
   def install
     cd "src" do
-      ENV["CC"] = ENV["MPICC"]
-      ENV["CXX"] = ENV["MPICXX"]
-
       system "./configure", "--prefix=#{prefix}",
-                            "--with-blas=yes",
-                            "--with-blas-libs=blas cblas",
-                            "--with-blas-lib-dirs=/usr/lib",
-                            "--with-lapack=yes",
-                            "--with-lapack-libs=lapack clapack f77lapack",
-                            "--with-lapack-lib-dirs=/usr/lib",
                             "--with-MPI",
                             "--enable-bigint"
```

Comment:
FWIW, I'm not convinced that we should be enabling big integers. This seems to unilaterally switch out integers for big integers in most locations. I also doubt that many macOS systems have the computing horsepower and/or memory to use big integers "for real". But, I can't find anything in the logs about why this was implemented.